portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java

```diff
@@ -0,0 +1,48 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.Map;
+import java.util.LinkedHashMap;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/chart-data")
+public class ChartServlet extends HttpServlet {
+
+  private LinkedHashMap<String, Integer> covidFloridaCases;
+  private LinkedHashMap<String, Integer> covidBrowardMiamiDadeCases;
```

Comment:
You may want to make the static type for these be Map<String, Integer> since them being LinkedHashMaps is only an implementation detail.

Reply:
from testing it out the line chart only works with linkedhashmaps